rocAL/rocAL/source/readers/image/file_source_reader.cpp

```diff
@@ -78,14 +78,14 @@ Reader::Status FileSourceReader::initialize(ReaderConfig desc)
         std::random_shuffle(_file_names.begin(), _file_names.end());
     _shuffle_time.end();
     return ret;
```

Comment:
Revert the line changes. Its adding one additional file.

Reply:
Reverted